ops/pebble.py

```diff
@@ -1314,7 +1314,17 @@ def __repr__(self):
 class NoticeType(enum.Enum):
     """Enum of notice types."""
 
+    CHANGE_UPDATE = 'change-update'
+    """Recorded whenever a change is updated, that is, when it is first
+    spawned or its status was updated. The key for change-update notices is
+    the change ID.
+    """
+
     CUSTOM = 'custom'
+    """A custom notice reported via the Pebble client API or "pebble notify".
```

Comment:
Should this be backticks?

```suggestion
    """A custom notice reported via the Pebble client API or `pebble notify`.
```